openedxscorm/scormxblock.py

```diff
@@ -183,7 +183,7 @@ def get_current_user(self):
     @staticmethod
     def resource_string(path):
         """Handy helper for getting static resources from our kit."""
-        data = pkg_resources.resource_string(__name__, path)
+        data = importlib_resources.files(__package__).joinpath(path).read_bytes()
```

Comment:
what if you use `__name__` instead of `__package__` here?

Reply:
Right `__name__` makes more sense here.